src/main/resources/api/service_openapi.yaml

```diff
@@ -459,6 +459,66 @@ components:
         hybridConnectionName:
           type: string
           description: The user-provided name to identify the Relay HybridConnection.
+    AzureContainerInstance:
+      type: object
+      required:
+        - resourceGroup
+        - containerGroupName
+        - containers
+        - operatingSystemTypes
```

Comment:
is `operatingSystemTypes` required if `image` is defined?

Reply:
Yes, 'operatingSystemTypes' and containers are [required](https://docs.microsoft.com/en-us/rest/api/container-instances/container-groups/create-or-update).